### DISCUSSION TITLE
Play sound when build finishes.

### DISCUSSION THREAD

**Discussion by pitermarx** [cake-build/cake#3782] [Ideas]
It's probably something better done on a addin. But what do you think?
Is there any xplat system sounds? I know of System.Media.SystemSounds, but I think this is windows only

**Comment by devlead** [comment]
I would suspect the best way xplat would be to just invoke a tool like the sox project `play` command or `mpg123` command-line tool using the built-in [StartProcess](https://cakebuild.net/api/Cake.Common/ProcessAliases/BB668900) alias.

**Reply by pitermarx** [reply]
Would you bundle sox inside an addin?

**Reply by devlead** [reply]
I would suspect not, but you would use tool resolution to resolve its path, as it's a native binary how it's acquired depends on the operating system.
